Dockerfile

```diff
@@ -42,8 +42,8 @@ RUN \
   git config --global user.name "Builder" && \
   git config --global user.email "builder@localhost"
 
-ARG BRVER="2021.02.1"
-ARG KVER="5.4.107"
+ARG BRVER="2021.02.3"
```

Comment:
[2021.05](https://github.com/buildroot/buildroot/releases) is also available, but I wasn't sure about patching it.